Automattic/wp-calypso
# Jetpack Cloud: Remove loading of part of layout on Cloud

#### Changes proposed in this Pull Request

This PR removes:
- Guided-tours
- nps-survay-notice
- support-article-dialog, 
- app-banner
As well as quering for editor preferences from Jetpack Cloud. 

This is done so that we remove requests that are not needed on Jetpack Cloud envoroment. And it improves the overall performance of Jetpack Cloud. 

#### Testing instructions

* Load up Jetpack Cloud Locally 
* Notice that the following requests are not there any more. 
( /calypso/evergreen/async-load-layout-guided-tours.ef869f801dbe0b7c0675.min.js )
( https://public-api.wordpress.com/rest/v1.2/nps ) 
( /calypso/evergreen/async-load-blocks-support-article-dialog.8d49900709146a27d5e5.min.js )
( /calypso/evergreen/async-load-blocks-app-banner.1044ed37e8119b18fe6e.min.js)
( /calypso/evergreen/async-load-blocks-app-banner.1044ed37e8119b18fe6e.min.css) 
( https://public-api.wordpress.com/wpcom/v3/sites/example.com/gutenberg)

* Load up calypso locally or and calypso live notice the following request still happen.
